cmd/object-api-utils.go

```diff
@@ -160,6 +157,28 @@ func IsValidObjectPrefix(object string) bool {
 	return true
 }
 
+// objectNameLength -check for the length of object name
+func objectNameLength(bucket, object string) error {
+	if len(object) > 1024 {
+		return ObjectNameTooLong{
+			Bucket: bucket,
+			Object: object,
+		}
+	}
+	return nil
+}
+
+// objectNamePrefixSlash -check for slash as prefix in object name
+func objectNamePrefixSlash(bucket, object string) error {
```

Comment:
Based on the function name should the return value be a boolean? Otherwise, do you think it should have a different name.

Reply:
changed